guides/doc-Upgrading_and_Updating/master.adoc

```diff
@@ -5,8 +5,26 @@ include::common/header.adoc[]
 
 = {UpgradingDocTitle}
 
+In this guide, the terms upgrade, update, and migrate have the following meanings:
+
+Upgrading::
+The process of advancing your {ProjectServer} and {SmartProxyServer} installations from a y-stream release to the next, for example {Project} {ProjectVersionPrevious} to {Project} {ProjectVersion}.
+For more information, see xref:upgrading_overview[].
+
+Updating::
+The process of advancing your {ProjectServer} and {SmartProxyServer} installations from a z-stream release to the next, for example {Project} {ProjectVersion}.0 to {Project} {ProjectVersion}.1.
+ifndef::foreman-deb,orcharhino[]
```

Comment:
In 3.1, `updating_project` seems only to be defined for Satellite, so that helps:
```suggestion
ifdef::satellite[]
```